### DISCUSSION TITLE
Add lateral genes page \ mark lateral gene in ggplot

### DISCUSSION THREAD

**Discussion by drorbar** [tanaylab/MCView#153] [Ideas]
First, when choosing a gene in the ggplot figures, you have no way of knowing if it's lateral or not - you can maybe go to the markers heatmap and add it and see the color, or find correlated gene with it and choose it as x\y and then see if you get lateral, but this seems like a bad option. We need to add (lateral) to the names of those lateral, when showing in the table the correlated genes 
Second, right now it's hard to share the lateral gene list we had with other groups\ projects, maybe a page with a list of all the lateral will be nice

**Comment by aviezerl** [comment]
Not sure I follow - which ggplot? on which tab?

**Reply by drorbar** [reply]
Let's say here - I have no way of knowing the MKI67 is lateral 
![image](https://github.com/tanaylab/MCView/assets/6944785/8dc3b65d-d7db-4ff3-9832-8520a83fc982)

What I can do is choose one of those genes which correlated with it, choose them and hope that MKI67 will be high enough in the list to see MKI67(lateral)

**Comment by aviezerl** [comment]
Done in 0.2.27
